pkg/maps/policymap/policymap.go

```diff
@@ -70,23 +70,21 @@ func (pe *PolicyEntry) String() string {
 // +k8s:deepcopy-gen=true
 // +k8s:deepcopy-gen:interfaces=github.com/cilium/cilium/pkg/bpf.MapKey
 type PolicyKey struct {
-	Identity         uint32
-	DestPort         uint16 // In network byte-order
-	Nexthdr          uint8
-	TrafficDirection uint8
+	Identity         uint32 `align:"sec_label"`
+	DestPort         uint16 `align:"dport"` // In network byte-order
+	Nexthdr          uint8  `align:"protocol"`
+	TrafficDirection uint8  `align:"egress"`
 }
 
 // PolicyEntry represents an entry in the BPF policy map for an endpoint. It must
 // match the layout of policy_entry in bpf/lib/common.h.
 // +k8s:deepcopy-gen=true
 // +k8s:deepcopy-gen:interfaces=github.com/cilium/cilium/pkg/bpf.MapValue
 type PolicyEntry struct {
-	ProxyPort uint16 // In network byte-order
-	Pad0      uint16
-	Pad1      uint16
-	Pad2      uint16
-	Packets   uint64
-	Bytes     uint64
+	ProxyPort uint16    `align:"proxy_port"` // In network byte-order
+	Pad       [3]uint16 `align:"pad"`
```

Comment:
@brb could you please use something similar to what we have for `pad3uint8`? Also, run `make generate-k8s-api` and will see why ;-)

Reply:
Aha. I will change it once we have resolved https://github.com/cilium/cilium/pull/9286#discussion_r328764148.